handlers/c_resources.go

```diff
@@ -20,10 +21,16 @@ func InitCResourceHandlers(cresourceRouter fiber.Router) {
 // @Tags         cluster-resources
 // @Produce      json
 // @Success      200  {array}  CResource
+// @Failure		 500
 // @Router       /api/cresources/ [get]
 func cresourceListHandler() func(c *fiber.Ctx) error {
 	return func(c *fiber.Ctx) error {
-		c.JSON([]CResource{})
+		cResources, err := db.GetAllResources()
+		if err != nil {
+			return c.SendStatus(500)
```

Comment:
Our error returns need some work. I think it's fine for this PR but we should afterwards open one for centralized error handling where we also return an error message.

Reply:
True. Added this as an issue.